src/const.ts

```diff
@@ -233,6 +235,10 @@ export interface Constant extends Required<ConstantProps> {
    * @private Callback to be called when the widget open state changes.
    */
   onWidgetOpenStateChange?: (params: OnWidgetOpenStateChangeParams) => void;
+  /**
+   * @public locale instance to be applied
+   */
+  locale: Locale;
```

Comment:
If you want to keep it public, could you move it to under `autoOpen` which is the section of public props? Also we need to add this to README prop table as well.